bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
Ok, so I agree that i18n is the more general definition and thus a better fit than l10n.

Regarding `i18n.InternationalizationProvider`: Note that there IS already a `i18n.I18nProvider` - which is used everywhere for translating texts. This provider already has a dependency on `LocaleProvider`. I wonder if we should merge the code of `LocaleProvider` into the `I18nProvider` and hence also add our new location there?
I would prefer having these features grouped in a single service and not having i18n, l10n and location as three different services that all depend on one another.
@maggu2810 As you introduced the LocaleProvider at the time - what's your opinion here?